docs/pages/setup/reference/backends.mdx

```diff
@@ -494,7 +494,11 @@ teleport:
 
 ### DynamoDB autoscaling
 
-When setting up DynamoDB it's important to set up backup and autoscaling. We make setup simpler by allowing AWS DynamoDB settings to be set automatically during Teleport startup.
+When setting up DynamoDB it's important to set up backup. Autoscaling is optional with Teleport,
+it is advised to collect some production usage data before enabling autoscaling.
+
+Autoscaling and backup can be configured on DynamoDB directly. We also make setup simpler by allowing AWS DynamoDB
+settings to be set automatically during Teleport startup.
```

Comment:
> We also make setup simpler by allowing AWS DynamoDB
settings to be set automatically during Teleport startup.

Maybe worth adding a link to where we explain how to do that.

Reply:
It's written below, I added a sentence.